src/request/components/RequestDetailPanelData.tsx

```diff
@@ -0,0 +1,48 @@
+import { MasterDetailTriPanel } from './MasterDetailTriPanel';
+import { RequestDetailPanelDataOperationCommandContainer } from '../containers/RequestDetailPanelDataOperationCommandContainer';
+import { RequestDetailPanelDataOperationTableContainer } from '../containers/RequestDetailPanelDataOperationTableContainer';
+
+import * as React from 'react';
+import Highlight = require('react-highlight');
+
+export interface IDataComponentProps {
+    totalOperationCount: number;
+}
+
+export class DataComponent extends React.Component<IDataComponentProps, {}> {
+    public render() {
+        return (
+            <div className='tab-data'>
+            {
+                this.props.totalOperationCount > 0
+                    ? <MasterDetailTriPanel masterPanel={this.renderMaster()} leftDetailPanel={this.renderLeftDetail()} leftDetailPanelTitle='Command' rightDetailPanel={this.renderRightDetail()} rightDetailPanelTitle='Summary' />
+                    : <div className='tab-section text-minor'>Could not find any data.</div>  
+            }
+            </div>
+        );
+    }
+    
+    private renderMaster() {
+        return (
+            <div className='tab-data-master'>
+                <div className='tab-data-operation-count'>{this.getTotalOperationCountText()}</div>
+                <br />
+                <RequestDetailPanelDataOperationTableContainer />
+            </div>
+        );
+    }
+    
+    private renderLeftDetail() {
+        return (
+            <RequestDetailPanelDataOperationCommandContainer />
+        )
+    }
+    
+    private renderRightDetail() {
+        return null;
+    }
+    
+    private getTotalOperationCountText() {
+        return this.props.totalOperationCount + (this.props.totalOperationCount === 1 ? ' Operation' : ' Operations');
+    }
```

Comment:
Again more generally, I wonder if its worth having an internal pluralization service that in case like this, you would pass in the word and the count and it would manage the lookup/conversation. This would also more things easier in the future when it comes to internationalization. But all of this is outside the scope of this PR.